docs/install/vendor-options.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Commercial Knative Offerings"
+linkTitle: "Knative Vendors"
+weight: 15
+type: "docs"
+---
+
+Knative has a rich community with many vendors participating, and many of those
+vendors offer commercial Knative offerings.  Please check with each of these
+vendors for what is or is not supported.
+
+<!-- TODO: Talk about criteria for inclusion here?  a la Conformance -->
+
+Here is a list of commercial Knative products (alphabetically):
+
+- [Google Cloud Run for Anthos](https://cloud.google.com/run/docs/gke/setup): <!-- TODO(ryangregg): add a description -->
+- IBM Kubernetes Service managed Knative add-on: <!-- TODO(duglin): add a description/link --> 
+- [Openshift Serverless](https://docs.openshift.com/container-platform/4.1/serverless/installing-openshift-serverless.html): <!-- TODO(bbrowning): add a description -->
```

Comment:
@bbrowning I'd like to include a short description of each of these (also please feel free to suggest a different name/link as appropriate.

Reply:
Let's change the OpenShift link to https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3/html-single/serverless_applications/index - newer than the docs linked and our official doc location that customers use. The description from that docs page that would fit the best here is

> OpenShift Serverless on OpenShift Container Platform enables stateful, stateless, and serverless workloads to all run on a single multi-cloud container platform with automated operations. Developers can use a single platform for hosting their microservices, legacy, and serverless applications.